File: Sreeya/Week 1/05-09-22/palindrome.cpp
class Solution {
public:
    bool isPalindrome(int x) {
    
    int sum=0,r;
    int t=x;    
    if (x<0) 
        return false;
    while (t>0) {
        r=t%10;
        sum=sum*10+r;
        t/=10;
    }
    return (sum==x);
        
    }
};

File: Pranchal/week1/palindrome.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        reversedNum =0
        if x%10 ==0 and x!=0: return False
        if x<0: return False
        while x>reversedNum:
            reversedNum = reversedNum*10 + x%10
            x = (int)(x/10)
        if (reversedNum == x) or ((int)(reversedNum/10) == x):return True
        else: return False


#https://leetcode.com/problems/palindrome-number/